src/app/helpers/environmentVariablesHelper.js

```diff
@@ -5,6 +5,8 @@ const packageObj = JSON.parse(fs.readFileSync('package.json', 'utf8'))
 
 let envVariables = {
 
+  APPLE_SIGNIN_KEY_URL: "https://appleid.apple.com/auth/keys",
```

Comment:
please place this in the bottom with info what is this variable and please do not hard code any value